src/fs/driver/devfs/devfs_oldfs.c

```diff
@@ -134,6 +134,8 @@ static struct inode *devfs_lookup(char const *name, struct inode const *dir) {
 	cdevtab = get_cdev_tab();
 	bdevs = get_bdev_tab();
 	for (i = 0; i < MAX_BDEV_QUANTITY; i++) {
+		if(!bdevs[i])
+			continue;
```

Comment:
Can you please follow our code style - https://github.com/embox/embox/wiki/Code-Style

```
if (...) {
    ...
}
```

Reply:
I am sorry, I will read your code style to avoid such errors in the future.